airbyte-integrations/connectors/source-jdbc-models/build.gradle

```diff
@@ -0,0 +1,24 @@
+import org.jsonschema2pojo.SourceType
```

Comment:
I was running into a weird issue while trying to nest this model inside of `airbyte-integrations/connectors/source-jdbc` so I pulled it out to be a top level connector while I was developing. I will fix the issue and nest everything that is currently in `airbyte-integrations/connectors/source-jdbc-models` in `airbyte-integrations/connectors/source-jdbc/models`